tests/go.mod

```diff
@@ -1,16 +1,16 @@
 module github.com/kubearmor/KubeArmor/tests
 
-go 1.21
+go 1.22
```

Comment:
kubearmor-client is using go1.22, so tests also. ig that's why snyk test failed. should I revert to go 1.21 for kubearmor-client or what?

Reply:
Oof... we tried using go 1.22 in KubeArmor earlier but snyk was not working with it for some reason. We concluded that snyk didn't support it back then.
IG now support for 1.22 is there. So this should be fine.

Reply:
@Aryan-sharma11 can you check why snyk (nyrahul) is failing?

Reply:
` Introduced through: github.com/kubearmor/KubeArmor/tests@0.0.0 › github.com/kubearmor/kubearmor-client@v1.2.0 › helm.sh/helm@v3.14.2`
but there's no version containing a fix for this vulnerability.

Reply:
Gotcha. I think we can update go in "test" package for now, updates to other modules can be handled in a different PR.